product_variant_sale_price/__manifest__.py

```diff
@@ -1,14 +1,15 @@
 # Copyright 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
 # Copyright 2018 Fabien Bourgeois <fabien@yaltik.com>
+# Copyright (C) 2020-Today: Druidoo (<https://www.druidoo.io>)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 {
     "name": "Product Variant Sale Price",
     "summary": "Allows to write fixed prices in product variants",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Product Management",
     "website": "https://odoo-community.org/",
-    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "author": "Tecnativa, " "Odoo Community Association (OCA), Druidoo",
```

Comment:
Unfortunately, migrating a module does not make us an author.
Please remove this :)